Order of ssl and crypto is wrong (first ssl, then crypto)
--- util/BUILD.gn
+++ util/BUILD.gn
@@ -402,8 +402,8 @@
         deps += [ "//third_party/boringssl" ]
       } else {
         libs = [
-          "crypto",
           "ssl",
+          "crypto",
         ]
       }
     }
@@ -670,8 +670,8 @@
         deps += [ "//third_party/boringssl" ]
       } else {
         libs = [
-          "crypto",
           "ssl",
+          "crypto",
         ]
       }
     }
